corgidrp/detector.py

```diff
@@ -377,7 +377,7 @@ def imaging_slice(obstype, frame, detector_regions=None):
     return sl
 
 def flag_cosmics(cube, fwc, sat_thresh, plat_thresh, cosm_filter, cosm_box,
-                   cosm_tail, mode='image'):
+                   cosm_tail, mode='image', detector_regions=None, obstype='SCI'):
```

Comment:
doesn't we call it `arrtype` instead of `obstype` since it checks against `arrtype`? to avoid future confusion